packer/install-dependencies.sh

```diff
@@ -95,7 +95,7 @@ for TILLER_VERSION in ${TILLER_VERSIONS}; do
     pullContainerImage "docker" "gcr.io/kubernetes-helm/tiller:v${TILLER_VERSION}"
 done
 
-CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.2 1.3.4 1.3.3 1.2.2 1.1.2"
+CLUSTER_AUTOSCALER_VERSIONS="1.13.1 1.12.2 1.3.7 1.3.3 1.2.2 1.1.2"
```

Comment:
Let's feel free to *add* component versions to VHD but retain old versions as well.

Reply:
There are no references to cluster-autoscaler 1.3.4 any longer, so I dropped it from the VHD list.

Reply:
Actually I guess 1.3.3 is also unused now.

Reply:
I removed the removal--we need to audit AKS as well for usage before aging things out of the VHD image.